### PR TITLE
Updates in Partner Program API

### PR DESCRIPTION
**Pull request by skix123**
Please do not merge it yet, PR made just for live preview

### CODE REVIEW

**Comment by skix123** [comment]
Feel free to merge and publish the changes

File: content/monetization/partner-program-api/index.mdx [hunk 1]
```diff
@@ -1398,23 +1404,27 @@ curl --request GET \
...
+- [_only for `livechat` product_] `coupon_id` - only for the `client` payment origin - the ID of the coupon assigned to a Partner (will apply a discount)
 
 #### Response
 
 - `201 - Created` - success, see example
 - `400 - Bad Request` - missing or incorrect parameters
 - `401 - Unauthorized` - a missing or incorrect authorization header
 - `403 - Forbidden` - not allowed to use the given `coupon_id`
-- `409 - Conflict` - the given `client_email` already has a LiveChat license
+- `409 - Conflict` - the given `client_email` already has a license for specified `product`
```
Comment (ale-kacperczyk) [inline content/monetization/partner-program-api/index.mdx]:
```suggestion
- `409 - Conflict` - the given `client_email` already has a license for the specified `product`
```

File: content/monetization/partner-program-api/index.mdx [hunk 2]
```diff
@@ -1607,13 +1603,62 @@ curl --request GET \
...
+
+
 <Section>
 
 <Text>
 
 ## Get License Operators
 
-`GET /v2/partners/solutions/licenses/<license_id>/operators` - get the list of operators
+`GET /v2/partners/solutions/licenses/livechat/<license_id>/operators` - get the list of LiveChat license's operators
```
Comment (ale-kacperczyk) [inline content/monetization/partner-program-api/index.mdx]:
```suggestion
`GET /v2/partners/solutions/licenses/livechat/<license_id>/operators` - get the list of a LiveChat license's operators
```

File: content/monetization/partner-program-api/index.mdx [hunk 3]
```diff
@@ -1925,7 +1970,7 @@ curl --request GET \
 
 ## Close Subscription
 
-`DELETE /v2/partners/solutions/licenses/<license_id>/subscription` - close a subscription
+`DELETE /v2/partners/solutions/licenses/livechat/<license_id>/subscription` - close a subscription for LiveChat license
```
Comment (ale-kacperczyk) [inline content/monetization/partner-program-api/index.mdx]:
```suggestion
`DELETE /v2/partners/solutions/licenses/livechat/<license_id>/subscription` - close a subscription for a LiveChat license
```

File: content/monetization/partner-program-api/index.mdx [hunk 4]
```diff
@@ -1959,7 +2004,7 @@ curl --request GET \
 
 ## Update License Billing
 
-`PUT /v2/partners/solutions/licenses/<license_id>/billing` - update the subscription billing information
+`PUT /v2/partners/solutions/licenses/livechat/<license_id>/billing` - update the subscription billing information for LiveChat license
```
Comment (ale-kacperczyk) [inline content/monetization/partner-program-api/index.mdx]:
```suggestion
`PUT /v2/partners/solutions/licenses/livechat/<license_id>/billing` - update the subscription billing information for a LiveChat license
```

File: content/monetization/partner-program-api/index.mdx [hunk 5]
```diff
@@ -1501,28 +1499,26 @@ curl --request GET \
...
-      "commission_percent": 0,
-      "expired": false,
+      "payment_origin": "partner",
       "blocked": false,
-      "data_center": "dal"
-    }
+      "active": true
+    },
+	{
+	  "license": "789",
```
[the remark this answers is not in the document]
Comment (skix123) [reply content/monetization/partner-program-api/index.mdx]: Fixed

File: content/monetization/partner-program-api/index.mdx (hunk 1 above)
Comment (skix123) [reply content/monetization/partner-program-api/index.mdx]: OK

File: content/monetization/partner-program-api/index.mdx (hunk 2 above)
Comment (skix123) [reply content/monetization/partner-program-api/index.mdx]: OK

File: content/monetization/partner-program-api/index.mdx (hunk 3 above)
Comment (skix123) [reply content/monetization/partner-program-api/index.mdx]: OK

File: content/monetization/partner-program-api/index.mdx (hunk 4 above)
Comment (skix123) [reply content/monetization/partner-program-api/index.mdx]: OK

**Pull request merged**